### ISSUE TITLE
Triggered by webhook

### ISSUE THREAD

**Issue by jwillmer** [aelassas/Wexflow#302]
Is it possible to trigger a workflow by a webhook? The possibilities of Wexflow are awesome but most of the time our processes get started by other systems that send a webhook.

**Comment by aelassas** [comment]
You can use [Wexflow API](https://github.com/aelassas/Wexflow/wiki/RESTful-API) to trigger workflows.

**Comment by jwillmer** [comment]
yes, but can I send a (POST) webhook to a Wexflow endpoint to start it (and use the information (JSON body) of the request in my workflow)?

**Comment by aelassas** [comment]
You can send a POST http://localhost:8000/wexflow/start?w={id} to start a workflow but it doesn't return a JSON response.

**Comment by jwillmer** [comment]
shall I create a feature request or is this something the engine does not focus on?

**Comment by aelassas** [comment]
You can create a feature request.
